src/js/tabs/setup.js

```diff
@@ -237,6 +237,7 @@ TABS.setup.initialize = function (callback) {
             if (semver.gte(CONFIG.apiVersion, "1.42.0")) {
                 disarmFlagElements.splice(disarmFlagElements.indexOf('THROTTLE'), 0, 'CRASH');
                 disarmFlagElements = disarmFlagElements.concat(['REBOOT_REQD']);
+                disarmFlagElements = disarmFlagElements.concat(['DSHOT_BITBANG']);
```

Comment:
I think is better to join the concat to the earlier element:
```
disarmFlagElements = disarmFlagElements.concat(['REBOOT_REQD'],
                                               ['DSHOT_BITBANG']);
```
I know we have code in both ways but I think is cleaner like this.

Reply:
Ok.

Reply:
Both are the same 😋 or at least I don't see difference.

Reply:
Well the difference is:
```
...disarmFlagElements.concat(['REBOOT_REQD'],['DSHOT_BITBANG']);
```
vs.
```
...disarmFlagElements.concat(['REBOOT_REQD', 'DSHOT_BITBANG']);
```
Maybe doesn't matter and I should probably just be consistent with the other existing code.